core-developers/committing.rst

```diff
@@ -80,21 +80,14 @@ to enter the public source tree. Ask yourself the following questions:
 Updating NEWS and What's New in Python
 --------------------------------------
 
-Almost all changes made to the code base deserve an entry in ``Misc/NEWS.d``.
-If the change is particularly interesting for end users (e.g. new features,
-significant improvements, or backwards-incompatible changes), then an entry in
-the ``What's New in Python`` document (in ``Doc/whatsnew/``) should be added
-as well. Changes that affect documentation only generally do not require
-a ``NEWS`` entry.
+Changes that entail NEWS entries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Almost all changes made to the code base deserve an entry in ``Misc/NEWS.d``.
 There are two notable exceptions to this general principle, and they
```

Comment:
I would say that minor changes (like typo fixes, some changes to documentation and tests, etc) are another notable exception.  These are mentioned below, but I find it a bit weird that it mentions two "notable" exception (that are somewhat uncommon) and then adds more exception afterwards.

IOW, I would just say something like: "NEWS entries are not needed for: <list of exceptions>"

Reply:
Yeah, I think the text is easier if we collect the exceptions in a single place. I'll give it a go.

Reply:
How about this (diff against PR):

```diff
diff --git a/core-developers/committing.rst b/core-developers/committing.rst
index 8d5e2d4..4e11ed5 100644
--- a/core-developers/committing.rst
+++ b/core-developers/committing.rst
@@ -83,11 +83,17 @@ Updating NEWS and What's New in Python
 Changes that require NEWS entries
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Almost all changes made to the code base deserve an entry in ``Misc/NEWS.d``.
-There are two notable exceptions to this general principle, and they
-both relate to changes that already have a ``NEWS`` entry,
-or changes that have not yet been included in any formal release
-(including alpha and beta releases):
+Most changes made to the code base deserve an entry in :cpy-file:`Misc/NEWS.d`,
+except for the following:
+
+* documentation changes in general
+* test changes in general
+* strictly internal changes with no user-visible effects
+* changes that already have a ``NEWS`` entry
+* changes that have not yet been included in any formal release
+  (including alpha and beta releases)
+
+For the latter two exceptions, note the following:
 
 #. **If a change is reverted prior to release**, then the corresponding
    entry is simply removed. Otherwise, a new entry must be added noting
```

Reply:
That sounds great!

Reply:
Much better. Let's use "last two" instead of "latter two".

https://www.merriam-webster.com/grammar/former-and-latter-usage

Remove "in general" from:

```
* documentation changes in general
* test changes in general
```

---

This one is confusing, does it more specifically mean _reverts_ rather than _changes_?

```
* changes that have not yet been included in any formal release
  (including alpha and beta releases)
```